helpers/elastic.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+ELASTIC_OPENDISTRO=$(echo $ELASTIC_OPENDISTRO | tr '[:upper:]' '[:lower:]')
+
```

Comment:
It could probably be simplified by something like:

```
auth=""
if [ "$ELASTIC_OPENDISTRO" = "true" ]; then
  auth=" --user ${ELASTIC_USER:-'admin'}:${ELASTIC_PASSWORD:-'admin'}"
fi
curl --silent $auth --fail localhost:9200/_cluster/health
```